assets/js/app/LocusZoom.js

```diff
@@ -245,42 +245,42 @@ LocusZoom.prettyTicks = function(range, clip_range, target_tick_count) {
  * @returns {Promise}
  */
 LocusZoom.createCORSPromise = function (method, url, body, headers, timeout) {
-    var response = Q.defer();
-    var xhr = new XMLHttpRequest();
-    if ('withCredentials' in xhr) {
-        // Check if the XMLHttpRequest object has a "withCredentials" property.
-        // "withCredentials" only exists on XMLHTTPRequest2 objects.
-        xhr.open(method, url, true);
-    } else if (typeof XDomainRequest != 'undefined') {
-        // Otherwise, check if XDomainRequest.
-        // XDomainRequest only exists in IE, and is IE's way of making CORS requests.
-        xhr = new XDomainRequest();
-        xhr.open(method, url);
-    } else {
-        // Otherwise, CORS is not supported by the browser.
-        xhr = null;
-    }
-    if (xhr) {
-        xhr.onreadystatechange = function() {
-            if (xhr.readyState === 4) {
-                if (xhr.status === 200 || xhr.status === 0 ) {
-                    response.resolve(xhr.response);
-                } else {
-                    response.reject('HTTP ' + xhr.status + ' for ' + url);
+    return new Promise(function (resolve, reject) {
```

Comment:
Reminder to self: add a deprecation warning to this function. We'll preserve it for custom deployments, but most new usages can probably be served by the built in `fetch`.